tests/basic_relay.go

```diff
@@ -114,8 +114,8 @@ func BasicRelay(network interfaces.LocalNetwork) {
 	relayerIDA := database.CalculateRelayerID(subnetAInfo.BlockchainID, subnetBInfo.BlockchainID, database.AllAllowedAddress, database.AllAllowedAddress)
 	relayerIDB := database.CalculateRelayerID(subnetBInfo.BlockchainID, subnetAInfo.BlockchainID, database.AllAllowedAddress, database.AllAllowedAddress)
 	// Modify the JSON database to force the relayer to re-process old blocks
-	jsonDB.Put(relayerIDA, database.LatestProcessedBlockKey, []byte("0"))
-	jsonDB.Put(relayerIDB, database.LatestProcessedBlockKey, []byte("0"))
+	_ = jsonDB.Put(relayerIDA, database.LatestProcessedBlockKey, []byte("0"))
+	_ = jsonDB.Put(relayerIDB, database.LatestProcessedBlockKey, []byte("0"))
```

Comment:
Let's check the returned errors here, else leave it as is.

Reply:
This sent me down a small rabbit hole... These were actually returning an error because the config wasn't validated, the `SupportedDestination` fields weren't populated, so `database.NewJSONFileStorage` wasn't populated with the correct relayer IDs.

I think it might make sense to add a `validated` field to the config struct, so that certain functions can check if the config has been validated before operating on it.

Reply:
Rather than adding a field that's unused in the main application, I'd prefer to instead make sure `Config.Validate` is idempotent, and call it here in the test. On inspection, I think it should already be idempotent.